code-of-conduct-faq.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Code of Conduct FAQ
+slug: code-of-conduct-faq
+---
+
+# Code of Conduct FAQ
+
+This document is for explaining why and how some decisions were made regarding [the SustainOSS code of conduct (CoC)](/code-of-conduct). An extra FAQ page is easier for us to update, as it does not require going through the entire review process for the CoC, and it makes it easier for us to explain some decisions without having to wrangle the wording to fit them into the CoC appropriately. Here's [an example](https://docs.fedoraproject.org/en-US/project/coc-clarifying/) of a similar document from Fedora.
+
+Do you have questions about the CoC, or this document? Send them along to [coc@sustainoss.org](mailto:coc@sustainoss.org). If you could specify that you want questions to be added to this document, we would appreciate it. Let's build this together.
+
+
+## The Questions
+
+#### When can I expect a response from the CoC committee?
+
+Within three to five days. Committee members can elect a stand-in if they know they will be AFK or unable to respond quickly to issues.
+
+#### When will you update the Code of Conduct, or check-in on how it is working?
+
+Our goal is to do this initially every three months. There will be an opportunity for the community to weigh in on their experience.
+
+#### Do you want comments on your CoC? When?
+
+We would love comments! We accept them on a rolling basis. Send a note to [coc@sustainoss.org](mailto:coc@sustainoss.org).
+
+#### Why did you make your own CoC? Why didn't you use X instead?
+
+We wanted to make a CoC that honoured our values, and we found many excellent examples of Codes of Conduct which we could have used. Ultimately, we decided to make our own CoC to work in both online and offline spaces. This decision was made by the lazy consensus of the SustainOSS organising committee. We didn't feel that any other CoC fit our needs perfectly.
```

Comment:
I believe we landed on the language of 'rough consensus' over 'lazy consensus' in our governance conversation. Small difference, but I think it's good to be consistent.

Reply:
Good catch! Fixed.